### PR TITLE
Add requirements.txt to official.

### PR DESCRIPTION
**Pull request by robieta**
Now that we have dependencies beyond what TensorFlow already pulls in, we may as well put a bit of formalism to help people out.

### CODE REVIEW

File: official/utils/logging/logger.py [hunk 1]
```diff
@@ -16,9 +16,7 @@
 """Logging utilities for benchmark.
 
 For collecting local environment metrics like CPU and memory, certain python
-packages need be installed. Run the following commands for dependency packages:
-  > pip install --upgrade py-cpuinfo
-  > pip install --upgrade psutil
+packages need be installed. See README for details.
```
Comment (karmel) [inline official/utils/logging/logger.py]: No reason to make comments less explicit here.

File: official/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+psutil>=5.4.3
+py-cpuinfo>=3.3.0
```
Comment (karmel) [inline official/requirements.txt]: Should tf-nightly be listed here too? I don't know if there's a way to do a conditional on gpu.

**Comment by karmel** [comment]
P.S. I assume you tested that this works, as I did not.

**Comment by robieta** [comment]
Yes. Knowing me that is a perfectly reasonable question.

**Pull request merged**